cdap-common/src/test/java/co/cask/cdap/common/test/AppJarHelper.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright © 2015-2016 Cask Data, Inc.
+ * Copyright © 2016 Cask Data, Inc.
```

Comment:
I think it's done by IntelliJ when I move the file from `src/main/java` to `src/main/test`.